django/librarian/utils/process_engine.py

```diff
@@ -514,7 +518,7 @@ def _pdf_to_html_using_azure(content):
     result = poller.result()
 
     num_pages = len(result.pages)
-    # TODO: create cost object based on number of pages
+    cost = Cost.objects.new(cost_type="doc-ai-prebuilt", count=num_pages)
```

Comment:
Form recognizer aka Document AI cost. This works regardless of how this is invoked, i.e. file summarization or librarian upload.